src/Software.php

```diff
@@ -963,6 +963,8 @@ public function showMergeCandidates()
         foreach ($iterator as $data) {
             $software->getFromDB($data["id"]);
             $entries[] = [
+                'itemtype' => 'Software',
```

Comment:
We try to prevent usage of "hardcoded" classes:
```suggestion
                'itemtype' => Software::class,
```